lib/movie/screens/movie_details_screen/movie_details_screen.dart

```diff
@@ -0,0 +1,324 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:get_it/get_it.dart';
+
+import 'package:movie/config.dart';
+import 'package:movie/infrastructure/theme/theme_extensions.dart';
+import 'package:movie/movie/domain/movie.dart';
+import 'package:movie/movie/screens/movie_details_screen/cubit/movie_details_cubit.dart';
+import 'package:movie/utils/date_time_formatting_extension.dart';
+
+// ---Texts---
+const _kScore = 'Score:';
+const _kRating = 'Rating:';
+const _kReleaseDate = 'Release Date:';
+const _errorText = 'Error';
+
+// ---Parameters---
+const _kPadding = 15.0;
+const _kHeight = 30.0;
+const _kWidth = 20.0;
+const _kFontSize = 22.0;
+const _kFontSizeText = 18.0;
+const _kHeightContainer = 280.0;
+const _kWidthContainer = 200.0;
+const _kRadius = 20.0;
+const _kWithOpacity = 0.8;
+const _kSigmaXY = 10.0;
+
+class MovieDetailsScreen extends StatefulWidget {
+  static const _routeName = '/movie-details-screen';
+
+  static PageRoute<MovieDetailsScreen> route(Movie movie) {
+    return MaterialPageRoute(
+      settings: const RouteSettings(name: _routeName),
+      builder: (context) {
+        return BlocProvider(
+          create: (context) => MovieDetailsCubit(movie),
+          child: const MovieDetailsScreen(),
+        );
+      },
+    );
+  }
+
+  const MovieDetailsScreen({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  State<MovieDetailsScreen> createState() => _MovieDetailsScreenState();
+}
+
+class _MovieDetailsScreenState extends State<MovieDetailsScreen> {
+  MovieDetailsCubit get movieCubit =>
+      BlocProvider.of<MovieDetailsCubit>(context);
+
+  @override
+  void initState() {
+    movieCubit.getMovie();
+    super.initState();
+  }
+
+//
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<MovieDetailsCubit, MovieDetailsState>(
+      builder: (context, state) {
+        if (state is MovieDetailsLoading) {
+          return Scaffold(
+            body: Stack(
+              children: [
+                _BackgroundImage(posterPath: movieCubit.movie.posterPath),
+                const Center(
+                  child: CircularProgressIndicator(
+                    backgroundColor: Colors.transparent,
+                    color: Colors.white,
+                  ),
+                ),
+              ],
+            ),
+          );
+        }
+        if (state is MovieDetailsSuccess && state.movie != null) {
+          final Movie movie = state.movie!;
+          return Scaffold(
+            backgroundColor: Colors.transparent,
+            appBar: AppBar(
+              title: Text(movieCubit.movie.title),
+            ),
+            body: SafeArea(
```

Comment:
remove `SafeArea` or add parameter `bottom: false` to fix it

<img width="521" alt="Screenshot 2022-03-31 at 19 21 49" src="https://user-images.githubusercontent.com/20111049/161103367-833d683c-78a7-4f13-85a6-369e2dcf8e70.png">